lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Appeals Extra Conditions
+  As a beta LPA I want to inform the Planning Inspectorate of any extra conditions I wish to be attached to the appeal decision (over and above the standard ones)
+So that the Inspector can decide whether or not to attach them to their decision.
+
+Scenario: AC01 - User navigates to 'Do you have extra conditions' page
+  Given The Householder planning appeal questionnaire page is presented
```

Comment:
Given the householder planning appeal questionnaire task list is presented

Reply:
Edited and matching step function edited.